demo/package.json

```diff
@@ -15,13 +18,36 @@
       "NODE_ENV": "production"
     }
   },
+  "babel": {
+    "presets": [
+      "es2015",
+      "react",
+      "stage-2"
+    ]
+  },
   "dependencies": {
     "cf-graphql": "^0.4.2",
     "cors": "^2.8.3",
     "express": "^4.15.3",
-    "express-graphql": "^0.6.6"
+    "express-graphql": "^0.6.6",
+    "prop-types": "^15.5.10",
+    "react": "^15.6.1",
+    "react-apollo": "^1.4.2",
+    "react-dom": "^15.6.1"
   },
   "devDependencies": {
```

Comment:
Please `rm -rf node_modules && npm i` to check if all deps are saved in the `package.json`.